free-1.md

```diff
@@ -10,41 +10,120 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+My first idea is for a DSL that allows the user to create simple looping 
```

Comment:
"Was there a need that was similar to one you thought of? If so, how are your thoughts about the problems similar to and different from your partners?"

I would personally really benefit from a DSL that does this! However, the three ideas I thought of were all music-related ideas. However, I think we both aim to make the creative process easier for users!